modules/react/select/stories/examples/WithIcons.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {FormField} from '@workday/canvas-kit-react/form-field';
+import {Select, useSelectModel} from '@workday/canvas-kit-react/select';
+import {Flex} from '@workday/canvas-kit-react/layout';
+import {
+  activityStreamIcon,
+  avatarIcon,
+  uploadCloudIcon,
+  userIcon,
+} from '@workday/canvas-system-icons-web';
+
+export const customOptions = [
+  {id: 'Activity Stream', icon: activityStreamIcon},
+  {id: 'Avatar', icon: avatarIcon},
+  {id: 'Upload Cloud', icon: uploadCloudIcon},
+  {id: 'User', icon: userIcon},
+];
+
+export const WithIcons = () => {
+  const model = useSelectModel({
+    items: customOptions,
+  });
+  const currentItem = model.navigation.getItem(model.state.selectedIds[0], model);
+  return (
+    <Flex>
+      <Select model={model}>
+        <FormField label="Contact" inputId="with-icons-select">
+          <Select.Input
+            width="300px"
+            inputStartIcon={currentItem.value.icon}
```

Comment:
What do you think about replacing `currentItem` with `selectedItem` for clarity?
```suggestion
  const selectedItem = model.navigation.getItem(model.state.selectedIds[0], model);
  return (
    <Flex>
      <Select model={model}>
        <FormField label="Contact" inputId="with-icons-select">
          <Select.Input
            width="300px"
            inputStartIcon={selectedItem.value.icon}
```